packages/razzle/scripts/test.js

```diff
@@ -28,20 +26,26 @@ require('../config/env');
 const jest = require('jest');
 const argv = process.argv.slice(2);
 
-// Watch unless on CI or in coverage mode
-if (!process.env.CI && argv.indexOf('--coverage') < 0) {
+// Watch unless on CI, in coverage mode, or explicitly running all tests
+if (
+  !process.env.CI &&
+  argv.indexOf('--coverage') === -1 &&
```

Comment:
More readable to use `String.prototype.includes` ?

Reply:
That code is introduced by this commit actually: https://github.com/facebookincubator/create-react-app/commit/19e0bb1881e24fb1ee3fe421413d4d87e67f68dd

Maybe to avoid ES6 usage not sure but you are right